src/xcode/ENA/ENA/Source/Models/Profile/AntigenTestProfile.swift

```diff
@@ -35,7 +35,7 @@ struct AntigenTestProfile: Codable {
 		try container.encodeIfPresent(firstName, forKey: .firstName)
 		try container.encodeIfPresent(lastName, forKey: .lastName)
 		if let dateOfBirth = dateOfBirth {
-			try? container.encodeIfPresent(ISO8601DateFormatter.justDate.string(from: dateOfBirth), forKey: .dateOfBirth)
+			try? container.encodeIfPresent(ISO8601DateFormatter().string(from: dateOfBirth), forKey: .dateOfBirth)
```

Comment:
I think we can keep the `.justDate` formatter as we don't need the time information, same for the decoding below. It makes the date model a bit more precise

Reply:
Done